sim/dex.ts

```diff
@@ -690,7 +697,8 @@ class ModdedDex {
 			const spcDV = Math.floor(ivs.spa / 2);
 			return {
 				type: hpTypes[4 * (atkDV % 4) + (defDV % 4)],
-				power: Math.floor((5 * ((spcDV >> 3) + (2 * (speDV >> 3)) + (4 * (defDV >> 3)) + (8 * (atkDV >> 3))) + (spcDV % 4)) / 2 + 31),
+				power: Math.floor(
+					(5 * ((spcDV >> 3) + (2 * (speDV >> 3)) + (4 * (defDV >> 3)) + (8 * (atkDV >> 3))) + (spcDV % 4)) / 2 + 31),
```

Comment:
All the `Math.floor`s here can be `~~`, which is hopefully enough to fit.

Reply:
There's one `Math.floor(...)` on the too long line, and replacing it with `(~~...)` its still too long and also becomes obsfuscated (the parens are required here, but even without them its still too long).

Reply:
They should still be changed to `trunc`.